README.md

```diff
@@ -24,6 +24,7 @@
 ---
```

Comment:
did `deno fmt` format this file? lmao

Reply:
Yeah lol deno fmt now supports markdown files